cmd/cloud-controller-manager/gkenetworkparamsetcontroller.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	cloudprovider "k8s.io/cloud-provider"
+	networkclientset "k8s.io/cloud-provider-gcp/crd/client/network/clientset/versioned"
+	networkinformers "k8s.io/cloud-provider-gcp/crd/client/network/informers/externalversions"
```

Comment:
I can see this folder anywhere, has this been generated or I'm looking in the wrong place?

Reply:
this was added in a previous pr, so it wont show up here.